Gemfile

```diff
@@ -34,7 +34,7 @@ else
   gem 'gds-sso', '13.2.0'
 end
 
-gem 'airbrake', '~> 4.3.4'
+gem 'airbrake', github: 'alphagov/airbrake', branch: 'silence-dep-warnings-for-rails-5'
```

Comment:
I think it's a stopgap until we move off errbit and onto something else entirely.  Be totally worth having this discussion in a wider forum though.